client/apps/edit/components/content/article_layouts/video.jsx

```diff
@@ -146,6 +148,16 @@ export class EditVideo extends Component {
           <label>Video Published</label>
         </EditVideoPublished>
 
+        <EditVideoReleaseDate>
+          <label>Release Date</label>
+          <input
+            type='date'
+            className='bordered-input bordered-input-dark'
+            value={moment(media.release_date).local().format('YYYY-MM-DD')}
+            onChange={(e) => this.onMediaChange('release_date', e.target.value)}
```

Comment:
Almost all the dates in positron are saved as [ISO dates](https://github.com/artsy/positron/blob/master/client/apps/settings/client/curations/display/components/campaign.jsx#L6), and reconverted back to this format on the client.

Reply:
Good call!